java-extensions/hive-reader/pom.xml

```diff
@@ -32,11 +32,6 @@
             <artifactId>hadoop-common</artifactId>
         </dependency>
 
-        <dependency>
-            <groupId>org.apache.hadoop</groupId>
-            <artifactId>hadoop-aws</artifactId>
```

Comment:
you use aswsdk to replace this hadoop-aws?

Reply:
no.  hadoop-aws be put into the hadoop/common. but it can be loaded at runtime.